translations/package.json

```diff
@@ -4,7 +4,18 @@
   "description": "Translations for the integreat frontend projects",
   "main": "index.js",
   "private": true,
+  "scripts": {
+    "manage": "babel-node tools/manage.js"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.12.5"
+  },
   "devDependencies": {
+    "@babel/core": "7.12.10",
+    "@babel/node": "7.12.10",
+    "@babel/plugin-proposal-class-properties": "7.12.1",
```

Comment:
Are these really needed? At least the plugin should not be necessary (as it is used nowhere).